doc/en/components/charts/types/pie-chart.md

```diff
@@ -63,102 +63,6 @@ Do Not Use Pie Chart When:
 - You have more than 6 or 8 segments (high data volume) — consider a Bar, Line or Area chart if it works for your data story.
 - It would be easier for the viewer to perceive the value difference in a Bar chart.
 
-<!-- Blazor -->
-In order to use the $PlatformShort$ Pie Chart, the following module is first required to be added to your application:
-
-```razor
-PieChartModule.Register(IgniteUIBlazor);
-```
-<!-- end: Blazor -->
-
-<!-- Angular, React, WebComponents -->
-## How to Make $PlatformShort$ Pie Chart
```

Comment:
I think we need content about blazor module and these sections (except Usage section)
```
## How to Make $PlatformShort$ Pie Chart
### 1. Install $PlatformShort$ core package
### 2. Install $PlatformShort$ charts
### 3. Install Required Modules```